diff --git a/.env b/.env
index 2740c24..7e4d6b6 100644
--- a/.env
+++ b/.env
@@ -1 +1,2 @@
-REACT_APP_JSON_API="http://localhost:3005/"
\ No newline at end of file
+REACT_APP_JSON_API="http://localhost:3005/"
+ABOUT_BE_API="http://localhost:5000/"
\ No newline at end of file
diff --git a/db.json b/db.json
deleted file mode 100644
index dc44b70..0000000
--- a/db.json
+++ /dev/null
@@ -1,31 +0,0 @@
-{
-    "posts": [{
-        "id": 1,
-        "name": "Satheesh",
-        "email": "Sat@mail.com",
-        "adderss": "South street"
-    }, {
-        "id": 2,
-        "name": "John",
-        "email": "john@mail.com",
-        "adderss": "North street"
-    }, {
-        "id": 3,
-        "name": "Robert",
-        "email": "robert@mail.com",
-        "adderss": "East street"
-    }, {
-        "id": 4,
-        "name": "Mani",
-        "email": "mani@mail.com",
-        "adderss": "West street"
-    }],
-    "comments": [{
-        "id": 1,
-        "body": "some comment",
-        "postId": 1
-    }],
-    "profile": {
-        "name": "typicode"
-    }
-}
\ No newline at end of file
diff --git a/src/App.tsx b/src/App.tsx
index b8a67e6..1d747db 100644
--- a/src/App.tsx
+++ b/src/App.tsx
@@ -11,10 +11,10 @@ import Admission from "./pages/admission/Admission";
 import Contacts from "./pages/contacts/Contacts";
 import NewsandEvents from "./pages/news/NewsandEvents";
 import RegistrationForm from "./pages/registration-form/registration";
+import Header from "./component/header/Header";
 
 function App() {
   const auth = useSelector((state: any) => state.login.isAuthenticated);
-  console.log("auth", auth);
 
   const dispatch = useDispatch();
   useEffect(() => {
@@ -27,6 +27,7 @@ function App() {
       <Router>
         {/* <Header /> */}
         <Switch>
+          {/* <ProtectedRoute auth={auth} exact path="/" component={Header}/> */}
           <Route path="/registrationform" component={RegistrationForm}/>
           <ProtectedRoute auth={auth} exact path="/" component={Home} />
           <Route exact path="/login" component={Login} />
diff --git a/src/component/about/AboutSection.tsx b/src/component/about/AboutSection.tsx
index ff883de..fb29888 100644
--- a/src/component/about/AboutSection.tsx
+++ b/src/component/about/AboutSection.tsx
@@ -52,6 +52,7 @@ export default class AboutSection extends React.Component<{},any> {
           </div>
           <div className="About-des">
             <p>
+              
               A school is an educational environment where children go to learn
               from a teacher. Topics such as reading, writing and mathematics
               are central to education. Most of a student's time is spent in a
diff --git a/src/component/footer/Footer.tsx b/src/component/footer/Footer.tsx
index f67c94a..9ec178e 100644
--- a/src/component/footer/Footer.tsx
+++ b/src/component/footer/Footer.tsx
@@ -35,7 +35,6 @@ export default class Footer extends Component {
   render() {
     const { name, email, items } = this.state;
     // console.log(items,email,name)
-    console.log(items);
     return (
       <div className="foot">
         <div className="footer">
diff --git a/src/model.tsx b/src/model.tsx
index 3b1f386..d4a4c01 100644
--- a/src/model.tsx
+++ b/src/model.tsx
@@ -9,6 +9,12 @@ export type dataModelFeat = {
   title: string;
   description: string;
 };
+
+export type dataModelAbout = {
+  main_heading: string;
+  description: string;
+};
+
 export interface navModel  {
   error: string;
   loading: boolean;
@@ -21,23 +27,39 @@ export type featModel = {
   feat: dataModelFeat[];
 };
 
+export type aboutModel = {
+  loading: boolean;
+  error: string;
+  about: dataModelAbout[];
+  // reducer:any;
+};
+
 export interface loginModel  {
   isAuthenticated: string | boolean;
   user: {email:string, pass:string};
 
 };
 export type reduxModel = {
+  reducer: any;
   nav: navModel;
   feat: featModel;
+  about: aboutModel;
 };
 
 export type Actions =
   | { type: "FETCH_FEAT_REQUEST"; payload: { loading: boolean } }
   | { type: "FETCH_FEAT_SUCCESS"; payload: { Features: dataModelFeat[] } }
   | { type: "FETCH_FEAT_FAILURE"; payload: { error: string } }
+
+  | { type: "FETCH_ABOUT_REQUEST"; payload: { loading: boolean } }
+  | { type: "FETCH_ABOUT_SUCCESS"; payload: { Features: dataModelAbout[] } }
+  | { type: "FETCH_ABOUT_FAILURE"; payload: { error: string } }
+  | { type: "FETCH_ABOUT_DATA"; payload: { error: string } }
+
   | { type: "FETCH_NAV_REQUEST" }
   | { type: "FETCH_NAV_SUCCESS"; payload: { Nav: dataModelNav[] } }
   | { type: "FETCH_NAV_FAILURE"; payload: { error: string } };
+  
    
   // export type Success=
   // { type: "LOGIN_SUCCESS"; payload: { isAuthenticated: boolean } };
diff --git a/src/pages/about/About.tsx b/src/pages/about/About.tsx
index 6424248..a64683a 100644
--- a/src/pages/about/About.tsx
+++ b/src/pages/about/About.tsx
@@ -1,31 +1,47 @@
-import React from 'react'
+import React, { useEffect } from 'react'
 import Footer from '../../component/footer/Footer'
 import Header from '../../component/header/Header'
 import Nav from '../../component/nav/Nav'
+import { fetchAbout } from '../../redux/about/actions/aboutActions'
 import "./About.css"
+import { connect } from 'react-redux'
+import { reduxModel } from '../../model'
+
+
+function About({aboutData, fetchAbout}: any) {
+const data = aboutData?.message;
+console.log(data&&data[0],aboutData);
+
+  useEffect(() => {
+    fetchAbout()
+  }, [])
 
-export default function About() {
   return (
     <div>
         <Header/>
         <Nav />
-        
-      <h1 className="main_heading">Welcome to our School</h1>
-      <div className='main_details'>
-            <p>
-              A school is an educational environment where children go to learn
-              from a teacher. Topics such as reading, writing and mathematics
-              are central to education. Most of a student's time is spent in a
-              classroom. This is where 10 to 30 people sit to take part in
-              educational discussion. In the United States, the average number
-              of students per classroom in primary schools is 23.1.[1] The term
-              "school" is used for many educational environments--particularly
-              in North America. There are different types of schools: elementary
-              schools (primary in the UK), middle schools (secondary in the UK),
-              and so on.
-            </p>
-            </div>
-            <Footer/>
+    
+      <h1 className="main_heading">{data&&data[0].main_heading}</h1>
+      <div className='main_details'>{data&&data[0].description}</div>
+        <Footer/>
     </div>
   )
 }
+
+const mapStateToProps = (state:reduxModel) => {
+  console.log(state.about.about,"mmmmmmmmmmmmmmmmmmmmm");
+  
+  return {
+    aboutData: state.about.about
+    // aboutData: state.reducer.about
+  }
+}
+
+const mapDispatchToProps  = (dispatch: any)=> {
+ 
+  return{
+  fetchAbout:()=>dispatch(fetchAbout())
+}
+}
+
+export default connect(mapStateToProps, mapDispatchToProps)(About)
\ No newline at end of file
diff --git a/src/pages/admission/Admission.tsx b/src/pages/admission/Admission.tsx
index 03d609e..13b6756 100644
--- a/src/pages/admission/Admission.tsx
+++ b/src/pages/admission/Admission.tsx
@@ -8,7 +8,7 @@ export default function Admission() {
       <Header />
       <Nav />
       <h1 className="main_heading">Admission Details</h1>
-      <p className="main_details">
+      {/* <p className="main_details">
         Admissions are made by the Principal based on the norms of selection and
         availability of seats. A child seeking admission to the Silver Hills
         Public School should be introduced by the parent or guardian who is
@@ -46,7 +46,15 @@ export default function Admission() {
         ponytail with black hair ties. 3. Boys: Neatly combed short hair. The
         school reserves the right to ensure strict adherence to the prescribed
         hairstyle.
-      </p>
+      </p> */}
+
+      <div className="admission">
+        
+      </div>
+      
+
+
+
       <Footer />
     </div>
   );
diff --git a/src/pages/login/Login.tsx b/src/pages/login/Login.tsx
deleted file mode 100644
index 833a34e..0000000
--- a/src/pages/login/Login.tsx
+++ /dev/null
@@ -1,68 +0,0 @@
-import React, { useState } from "react";
-import "./Login.css";
-import { useSelector, useDispatch } from "react-redux";
-import { getLogin } from "../../redux/login/actions/loginActions";
-import { Link, useHistory } from "react-router-dom";
-import axios from "axios";
-
-
-const Login = () => {
-  const dispatch = useDispatch();
-  const history = useHistory();
-  const state = useSelector((state:any) => state.login.user);
-  const [email, setEmail] = useState("");
-  const [password, setPassword] = useState("");
-  // console.log("hoo", state)
-  const handleSubmit =  (e: { preventDefault: () => void; }) => {
-    e.preventDefault();
-    const formdata = {email:email, password:password}
-
-    axios.post("http://localhost:5000/api/v1/login", formdata)
-    .then(res=>localStorage.setItem('token', JSON.stringify(res.data.token)))
-
-  
-    if (email && password) {
-      if (state.email === email && state.pass === password){
-        dispatch(getLogin());
-        history.push("/")
-      }else{
-        alert("Credentials does not match")
-      }
-      // console.log(email, password);
-    } else {
-      alert("Wrong credentials");
-    }
-  };
-
-  return (
-    <div className="login">
-      <form className="login__form" onSubmit={handleSubmit}>
-        <h2>Login Page</h2>
-        <input
-          type="email"
-          placeholder="Enter Email Address"
-          value={email}
-          onChange={(e) => setEmail(e.target.value)}
-        />
-        <input
-          type="password"
-          placeholder="Enter Password"
-          value={password}
-          onChange={(e) => setPassword(e.target.value)}
-        />
-        <button type="submit" className="submit__btn">
-          Submit
-        </button>
-        <Link to="/registrationform">{"Register Now?"}</Link>
-      </form>
-      
-    </div>
-    
-  );
-};
-
-export default Login;
-function token(token: any) {
-  throw new Error("Function not implemented.");
-}
-
diff --git a/src/redux/features/actions/featActions.tsx b/src/redux/features/actions/featActions.tsx
index 0415088..f62941a 100644
--- a/src/redux/features/actions/featActions.tsx
+++ b/src/redux/features/actions/featActions.tsx
@@ -35,7 +35,6 @@ export const fetchFeat = () => {
 
       .then((response) => {
         const feat = response.data;
-        console.log("vvvvvvvvvvvvvv", feat);
         
         dispatch(fetchFeatSuccess(feat));
       })
diff --git a/src/redux/nav/actions/navActions.tsx b/src/redux/nav/actions/navActions.tsx
index 8e61ca2..50908f5 100644
--- a/src/redux/nav/actions/navActions.tsx
+++ b/src/redux/nav/actions/navActions.tsx
@@ -30,7 +30,7 @@ export const fetchNav = () => {
     dispatch(fetchNavRequest)
     axios
       .get(`${process.env.REACT_APP_JSON_API}nav`)
-      // .get("http://localhost:3005/nav")
+      
       .then((response) => {
         const nav = response.data;
         dispatch(fetchNavSuccess(nav))
diff --git a/src/redux/rootReducer.tsx b/src/redux/rootReducer.tsx
index fbb185b..4b41482 100644
--- a/src/redux/rootReducer.tsx
+++ b/src/redux/rootReducer.tsx
@@ -2,11 +2,13 @@ import {combineReducers} from "redux"
 import navReducer from "./nav/reducers/navReducer"
 import featReducer from "./features/reducers/featReducer"
 import authReducer from "./login/reducers/loginReducer"
+import aboutReducer from "./about/reducers/aboutReducer"
 
 const rootReducer = combineReducers({
     nav:navReducer,
     feat:featReducer,
     login:authReducer,
+    about:aboutReducer,
 })
 
 export default rootReducer
\ No newline at end of file
